roofit/roofitcore/inc/RooMinimizer.h

```diff
@@ -39,6 +39,13 @@ class RooRealVar;
 class RooArgSet;
 class RooPlot;
 class RooDataSet;
+#ifdef ROOFIT_MULTIPROCESS
+namespace RooFit {
+namespace TestStatistics {
+class LikelihoodGradientJob;
+}
+} // namespace RooFit
+#endif // ROOFIT_MULTIPROCESS
```

Comment:
```suggestion
namespace RooFit {
namespace TestStatistics {
class LikelihoodGradientJob;
}
} // namespace RooFit
```

`ROOFIT_MULTIPROCESS` is only defined when building ROOT, so the macros like this should not be used in public header files. Here, since you are only introducing a forward declaration, can we just always do it?